### DISCUSSION TITLE
Table pagination css styles not working

### DISCUSSION THREAD

**Discussion by AdamBrutsaert** [netbox-community/netbox#14076] [Help Wanted!]
Hello,

I have a problem with my tables that I render inside my plugin generic.ObjectView, the css styles for the bottom pagination isn't applied :

<img width="1165" alt="Capture d’écran 2023-10-18 à 18 43 32" src="https://github.com/netbox-community/netbox/assets/79027326/69d59a6c-aa83-4ba0-93fc-a0f8f34356f0">

I render them like so :

```html
{% extends 'generic/object.html' %}
{% load render_table from django_tables2 %}

{% block content %}
  ...
  <div class="row mt-3">
    <div class="col">
      <div class="card">
        <h5 class="card-header">Title</h5>
        <div class="card-body table-responsive">
          {% render_table table %}
        </div>
      </div>
    </div>
  </div>
{% endblock content %}
```

I do inherit my table classes from NetBoxTable and the Meta classes inside them from NetBoxTable.Meta, and the bottom pagination css styles does work inside the generic.ObjectListView.

I suppose it's because the styles for 'pagination' aren't loaded inside generic.ObjectView. Nonetheless, I don't know how I should fix that. I would be grateful if someone could help me 😊

**Comment by McGlovin1337** [comment]
Appreciate this is an old discussion, however I recently ran into the same issue and stumbled across this discussion hoping to find the answer.
In the end figured a solution for this myself.
When instantiating the table class in my `ObjectView` class, I added the `template_name` argument with a value of `django_tables2/bootstrap5-responsive.html`, e.g.

```python
table = MyTable(table_data, template_name='django_tables2/bootstrap5-responsive.html')
```
This gave me nicely styled page navigation for the table.
